tests/test_bugbear.py

```diff
@@ -336,6 +336,14 @@ def test_does_not_crash_on_tuple_expansion_in_except_statement(self):
         )
         BugBearVisitor(filename="<string>", lines=[]).visit(syntax_tree)
 
+    def test_does_not_crash_on_call_in_except_statement(self):
+        # akin to test_does_not_crash_on_tuple_expansion_in_except_statement
+        # see https://github.com/PyCQA/flake8-bugbear/issues/171
+        syntax_tree = ast.parse(
+            "foo = lambda: IOError\ntry:\n    ...\nexcept (foo(),):\n    ...\n"
```

Comment:
I too, don't like this autoformat ...

cc: @ichard26 @JelleZijlstra @zsol - I think this will get complaints ... thoughts?

Reply:
Aren't user splits preserved unless they violate the length limit according to the ESP rules?

Reply:
Maybe the issue is that the string put together isn't long enough to hit the line length limit, so we put it all together.

Reply:
I think this is similar to https://github.com/psf/black/issues/1540